apps/fulfillment/.env.template

```diff
@@ -1,3 +1,2 @@
-ORYX_FULFILLMENT_BACKEND_URL=http://localhost:3001
 ORYX_LABS=true
```

Comment:
I'd drop labs from the template, we don't use it and dont' have anything in labs for picking afaik.